### PR TITLE
PR: Add test for plugin building

### PR DESCRIPTION
**Pull request by rlaverde**

### CODE REVIEW

**Comment by goanpeca** [comment]
and leta run ciocheck on that created repo

File: circle.yml [hunk 1]
```diff
@@ -19,9 +19,17 @@ dependencies:
       source ci-helpers/travis/setup_conda_$TRAVIS_OS_NAME.sh &&
       export PATH="$HOME/miniconda/bin:$PATH" &&
       source activate test &&
-      conda install pytest &&
+      conda install pytest entrypoints &&
       conda install pytest-cookies -c conda-forge &&
-      conda install ciocheck -c conda-forge;
+      conda install versioneer -c conda-forge &&
+      conda install ciocheck -c conda-forge &&
+      pip install -U pip;
```
Comment (goanpeca) [inline circle.yml]: why ?

Comment (rlaverde) [reply circle.yml]: errors with entrypoints

`error: Could not find suitable distribution for Requirement.parse('entrypoints>=0.2.2')`

Comment (goanpeca) [reply circle.yml]: Why are you installing with pip here?? I dont understand

File: circle.yml [hunk 2]
```diff
@@ -19,9 +19,17 @@ dependencies:
       source ci-helpers/travis/setup_conda_$TRAVIS_OS_NAME.sh &&
       export PATH="$HOME/miniconda/bin:$PATH" &&
       source activate test &&
-      conda install pytest &&
+      conda install pytest entrypoints &&
       conda install pytest-cookies -c conda-forge &&
-      conda install ciocheck -c conda-forge;
+      conda install versioneer -c conda-forge &&
+      conda install ciocheck -c conda-forge &&
```
Comment (goanpeca) [inline circle.yml]: conda install pytest-cookies -c conda-forge &&
conda install ciocheck -c conda-forge
conda install versioneer -c conda-forge &&
conda install ciocheck -c conda-forge &&

Move all these to the spyder-ide channel, I will copy the packages for you

Comment (rlaverde) [reply circle.yml]: ok, thanks :smile:

File: circle.yml (hunk 1 above)
Comment (rlaverde) [reply circle.yml]: Me neither :roll_eyes: I found that solution googling and worked in my local environment.

Comment (goanpeca) [reply circle.yml]: Please remove that

File: circle.yml [hunk 3]
```diff
@@ -19,9 +19,16 @@ dependencies:
       source ci-helpers/travis/setup_conda_$TRAVIS_OS_NAME.sh &&
       export PATH="$HOME/miniconda/bin:$PATH" &&
       source activate test &&
-      conda install pytest &&
+      conda install pytest cookiecutter -c conda-forge &&
       conda install pytest-cookies -c conda-forge &&
-      conda install ciocheck -c conda-forge;
+      conda install versioneer -c conda-forge &&
+      conda install ciocheck -c conda-forge &&
+      conda update pip -c conda-forge;
```
Comment (goanpeca) [inline circle.yml]: Why this?

Comment (rlaverde) [reply circle.yml]: I'm just testing, I think that the version from default channel was causing  the error in previous builds, updating it to conda-forge channel seems to fix the error

Comment (goanpeca) [reply circle.yml]: Then I will copy to spyder-ide as well (use spyder-ide only)

Comment (rlaverde) [reply circle.yml]: Forget it, the error was missing entrypoints dependency.

**Comment by goanpeca** [comment]
I added the requested deps to spyder-ide channel

**Comment by goanpeca** [comment]
LGTM :-)

**Comment by rlaverde** [comment]
Merging, thanks @goanpeca :smile:

**Pull request merged**